### PR TITLE
Provide feedback if invalid behold is analyzed

### LINKED ISSUE #23: Provide feedback if invalid behold is analyzed
Right now if bot cannot identify 3 crew who make sense, it says/does nothing.

If the bot is confident it is looking at a behold, it should send a message encouraging user to try a different screenshot, and noting the limitations around low-height crew etc.

### PR DESCRIPTION
**Pull request by bethune-bryant**
Making the behold determination less restrictive and providing output if classification fails.

Fixes #23

### CODE REVIEW

File: src/commands/imageanalysis.ts [hunk 1]
```diff
@@ -32,8 +32,20 @@ export async function runImageAnalysis(message: Message, url: string, usedPrefix
...
 			);
-		} else if (data.beholdResult && isValidBehold(data.beholdResult, 5)) {
-			await calculateBehold(message, data.beholdResult, false, false);
+		} else if (data.beholdResult && isPossibleBehold(data.beholdResult, 5)) {
+			if (isValidBehold(data.beholdResult, 5)) {
+				await calculateBehold(message, data.beholdResult, false, false);
+			} else{
+				sendAndCache(
+					message,
+					"Sorry, the image appears to be a behold, but the crew cannot be identified. You can try submitting another screenshot, but there are some limitations related to low-height crew."
```
Comment (ineffyble) [inline src/commands/imageanalysis.ts:41]: Is it worth adding a line about the fuzz/light effect that can cause this?

Comment (bethune-bryant) [reply src/commands/imageanalysis.ts:41]: That makes sense, I can add that.
Are there any other cases we'd want to address?

File: src/commands/imageanalysis.ts [hunk 2]
```diff
@@ -32,8 +32,20 @@ export async function runImageAnalysis(message: Message, url: string, usedPrefix
...
+				);
+			}
+		} else {
+			sendAndCache(
+				message,
+				"Sorry, the image does not appear to be either a valid voyage or behold."
```
Comment (ineffyble) [inline src/commands/imageanalysis.ts:47]: The bot will send this for every single image in any channel it is in. Could be a lot of noise in some channels.

File: src/commands/imageanalysis.ts (hunk 1 above)
Comment (ineffyble) [reply src/commands/imageanalysis.ts:41]: Not that I can think of, it's nearly always short crew or fuzzyness.

File: src/commands/imageanalysis.ts (hunk 2 above)
Comment (bethune-bryant) [reply src/commands/imageanalysis.ts:47]: Ahh, I didn't realize that. Yeah, that would be too much.
I'll remove this and just leave the other message.

File: src/commands/imageanalysis.ts (hunk 1 above)
Comment (bethune-bryant) [reply src/commands/imageanalysis.ts]: That's updated, let me know if it sounds okay.

**Review by ineffyble — APPROVED** [review]
Looks good, thanks!

File: src/commands/imageanalysis.ts (hunk 1 above)
Comment (AlexCPU) [reply src/commands/imageanalysis.ts]: Possibly plug the behold tool on the website for these low-height crew cases?

Comment (ineffyble) [reply src/commands/imageanalysis.ts]: Ohhh, that's smart.

Comment (ineffyble) [reply src/commands/imageanalysis.ts]: Once I've confirmed it's working as-is I'll tweak the text.

**Pull request merged**